dpnp/backend/kernels/dpnp_krnl_random.cpp

```diff
@@ -1242,65 +1243,70 @@ void dpnp_rng_vonmises_large_kappa_c(void* result, const _DataType mu, const _Da
 
     Uvec = reinterpret_cast<_DataType*>(dpnp_memory_alloc_c(size * sizeof(_DataType)));
     Vvec = reinterpret_cast<_DataType*>(dpnp_memory_alloc_c(size * sizeof(_DataType)));
-
-    for (size_t n = 0; n < size;)
+    n = reinterpret_cast<size_t*>(dpnp_memory_alloc_c(sizeof(size_t)));
```

Comment:
this is quite strange (Make scalar as a array with one element).
I think it should be a scalar, not an array.

Reply:
scalar and this is the same. You just can not pass n to sycl region in other way.